get-journey-summary-sample.html

```diff
@@ -0,0 +1,51 @@
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=0;"/>
+    <!-- fix the issue where elements are too small on mobile devices -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
+
+    <!-- TODO please don't change this URL -->
+    <script src="https://expose-journey-store.dff2qf83gvw45.amplifyapp.com/idwise.min.js"></script>
+
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+    </style>
+</head>
+<body>
+<button onclick="getJourney()">Log Journey Summary to console</button>
+<br>
+<div id="idwise-mount"></div>
+
+<script>
+  let idwiseInstance;
+
+  IDWise.initialize({
+    clientKey: "REPLACE_WITH_YOUR_CLIENT_KEY", // Replace with your client key
+  })
+    .then(idwise => {
+      idwiseInstance = idwise;  // assign idwise instance to a global variable
+
+      idwise.startJourney({
+        mount: "#idwise-mount",
+        journeyDefinitionId: "REPLACE_WITH_YOUR_JOURNEY_DEFINITION_ID", // journey definition id
+        eventHandlers: {
+          onJourneyFinished: function () {
+            getJourney(); // call getSummary() to get the journey summary
```

Comment:
Why is this called getJourney can we call it getJourneySummary to be consistent with iOS and Android? and update the comment please
Also let's show in the example the main elements that exist in the summary and use them. One sample use case is to iterate over has_passed_rules for all steps to check if all steps were successful or some weren't